constants/levelDataType.ts

```diff
@@ -1,105 +1,129 @@
-export default class LevelDataType {
+export enum TileType {
+  Default = '0',
+  DefaultVisited = 'X',
+  Wall = '1',
+  Block = '2',
+  End = '3',
+  Start = '4',
+  Hole = '5',
+  Left = '6',
+  Up = '7',
+  Right = '8',
+  Down = '9',
+  UpLeft = 'A',
+  UpRight = 'B',
+  DownRight = 'C',
+  DownLeft = 'D',
+  NotLeft = 'E',
+  NotUp = 'F',
+  NotRight = 'G',
+  NotDown = 'H',
+  LeftRight = 'I',
+  UpDown = 'J',
+}
+
+export default class LevelUtil {
```

Comment:
since this is being refactored anyways, we should move all these functions to a separate file (helpers/tileTypeHelper.ts)

then we can rename this file to tileType.ts

Reply:
and then we can do:

```
enum TileType {
...
}

export default TileType;
```